src/@ionic-native/plugins/user-agent/index.ts

```diff
@@ -0,0 +1,73 @@
+import {Injectable} from '@angular/core';
+import {Plugin, Cordova, IonicNativePlugin} from '@ionic-native/core';
```

Comment:
Please add spaces between the import names and the brackets. A space after `{` and a space before `}`.